torchvision/models/swin_transformer.py

```diff
@@ -160,7 +160,20 @@ def shifted_window_attention(
     return x
 
 
+def _fix_window_and_shift_size(
+    input_hw: List[int], window_size: List[int], shift_size: List[int]
+) -> Tuple[List[int], List[int]]:
+    # Handle case where window_size is larger than input tensor
+    # Reference on the original implementation: https://github.com/microsoft/Swin-Transformer/blob/main/models/swin_transformer.py#L192-L195
+    for i in range(2):
```

Comment:
Does it make sense to not hard-code 2 but make it an argument? Do you think this is likely to be reused in 3d?